ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -111,21 +152,29 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'P4S Kothornos Kick',
       regex: /Kothornos Kick/,
-      beforeSeconds: 5,
+      beforeSeconds: 5.3,
       infoText: (data, _matches, output) => {
-        if (data.role === 'tank' || data.role === 'healer')
-          return;
-        return output.text!();
+        let jumpDir = '';
```

Comment:
I'm not sure this fixes whatever the issue was.  If `data.jumpDir1` is somehow not east or west, then should this trigger not print out anything? (or print out some other `Bait jump` with no dir output string? I'm not sure if grammatically it can just have a blank dir and have that work in all languages)

Reply:
Could be possible to move the {dir} somewhere else in that language such that it would make sense, but it's easier to just have a second output in that situation than require the extra thought.